modules/imaging_browser/php/NDB_Menu_Filter_imaging_browser.class.inc

```diff
@@ -149,7 +151,14 @@ class NDB_Menu_Filter_Imaging_Browser extends NDB_Menu_Filter
             'Links',
             'sessionID'
         );
-       
+
+        // Insert project column after DCCID, if useProject config is enabled
+        if ($config->getSetting('useProjects') === "true") {
```

Comment:
This should probably be a truthier comparison. It could also be in the config.xml as "1"